src/components/IdentifierCardEditing.vue

```diff
@@ -84,10 +86,11 @@
 </template>
 
 <script lang="ts">
-import { IdentifierTypeType } from '../types'
+import { IdentifierTypeType, linkInfoType } from '../types'
 import { computed, defineComponent, PropType } from 'vue'
 import { getMyErrors } from 'src/store/validator'
 import { identifierErrors } from 'src/identifier-errors'
+import SchemaGuideLink from 'components/SchemaGuideLink.vue'
```

Comment:
apparently we have defined `components` somewhere in the config, but I prefer to use

```suggestion
import SchemaGuideLink from 'src/components/SchemaGuideLink.vue'
```